python/cudf/cudf/_lib/lists.pyx

```diff
@@ -58,3 +61,27 @@ def explode_outer(Table tbl, int explode_column_idx, bool ignore_index=False):
         column_names=tbl._column_names,
         index_names=None if ignore_index else tbl._index_names
     )
+
+
+def sort_lists(Column col, bool ascending, object na_position):
+    if not isinstance(col.dtype, ListDtype):
+        raise TypeError("col is not a list column.")
```

Comment:
Given this in a very internal Cython implementation and only accessible to Python via the `.list` accessor which would validate this already, I think we can remove this from here.

Reply:
I think we do this for a few other list functions in here, could you fix all of them?

Reply:
I think there's a similar line at 31-32 of this file - should we remove that in this PR too?

Reply:
Yes please